trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py

```diff
@@ -445,14 +477,105 @@ def add_unknowns_and_residuals_to_segment(self, segment, initial_voltage = None,
         segment.state.conditions.propulsion.disc_loading               = 0. * ones_row(n_props)                 
         segment.state.conditions.propulsion.power_loading              = 0. * ones_row(n_props)
         segment.state.conditions.propulsion.propeller_tip_mach         = 0. * ones_row(n_props)
-        segment.state.conditions.propulsion.propeller_efficiency       = 0. * ones_row(n_props)        
+        segment.state.conditions.propulsion.propeller_efficiency       = 0. * ones_row(n_props)   
+        segment.state.conditions.propulsion.figure_of_merit            = 0. * ones_row(n_props)      
         
         # Ensure the mission knows how to pack and unpack the unknowns and residuals
         segment.process.iterate.unknowns.network  = self.unpack_unknowns
         segment.process.iterate.residuals.network = self.residuals        
 
         return segment
     
+
+    def add_tiltrotor_transition_unknowns_and_residuals_to_segment(self, segment, initial_voltage = None, 
```

Comment:
this should get doxygen tags as well as the ones above

Reply:
added doxygen tags for both functions